app/controllers/api/auth/stories_controller.rb

```diff
@@ -32,7 +38,12 @@ def filtered(resources)
   end
 
   def resources_base
-    @stories ||= current_user.approved_account_stories
+    # If there is a network_id specified, use that network
```

Comment:
This implies that the "network-visible stories" are always going to be a subset of `current_user.approved_account_stories`... right?  Or am I misreading that?